app/routes.js

```diff
@@ -51,27 +51,27 @@ module.exports = (app, passport, mongoose) => {
           });
       });
 
-    // app.post('/testpdf', (req, res) => {
-    //     let pdfs = [];
-    //
-    //     for (let i = 0; i < req.body.pdfs.length; i++) {
-    //       let str = req.body.pdfs[i].slice(24, req.body.pdfs[i].length);
-    //       pdfs.push({
-    //               path: 'localStorage/pdfs/' + req.body.pdfs[i],
-    //               name: str
-    //             });
-    //
-    //       // pdfs.push(req.body.pdfs[i]);
-    //     }
-    //
-    //     console.log(pdfs);
-    //     res.send('asdf');
-    //   });
+    let pdfs = [];
```

Comment:
Could not get this code to work. Please see attached screenshots http://imgur.com/a/5g0d6